Tests/ScipioKitTests/RunnerTests.swift

```diff
@@ -268,6 +269,78 @@ final class RunnerTests: XCTestCase {
         }
     }
 
+    func testBuildClangPackageWithCustomModulePath() async throws {
```

Comment:
I think we can test this in FrameworkModuleMapGeneratorTests as a more basic unit test.

Reply:
Thank you for your review.
I moved the test to FrameworkModuleMapGeneratorTests in the following commit.
[af9a873](https://github.com/giginet/Scipio/pull/235/commits/af9a873caa2cabd6f60b021c0024694171f56725)